_schema.graphql

```diff
@@ -6923,10 +6923,11 @@ type CreateOrderWithArtworkPayload {
 
 # Autogenerated input type of CreateSavedSearch
 input CreateSavedSearchInput {
-  attributes: SearchCriteriaAttributes
+  attributes: SearchCriteriaAttributes!
```

Comment:
@dimatretyak Note that `attributes` is now a required argument for the `createSavedSearch` mutation. We should already be consistently providing attributes whenever this mutation is called so I don't anticipate any problems with this change.